chore: setting prettier and update eslint default config

pmpwith2i opened the pull request:
# Description 
This PR introduces some changes to default configuration of eslint and also update packages configuration in order to use prettier and the formatter for tailwindCSS classes.
This PR will introduce a refactor of many files in order to fit the new style rules, nothing changed in the core of the application.

# Issue
This PR closes #291

pmpwith2i committed:
chore: setting eslint and prettier

pmpwith2i committed:
chore: remove .vscode

pmpwith2i commented:
Hi @DonnySolana, while using this scaffold I added some configurations and I think that could be useful for other users to use it. Feel free to remove/add rules on eslint or to deny this PR.
Thanks! ♥︎

DonnySolana commented:
hey thanks!  what will the refactor look like? If you do not mind doing it, it may make more sense to do it at once and make sure everything runs smooth.

pmpwith2i commented:
> hey thanks! what will the refactor look like? If you do not mind doing it, it may make more sense to do it at once and make sure everything runs smooth.

Hey, essentially most of the refactor would be only to fix code style like quotes and indent. Then there are some typing issues to solve, I’ll do it in those days:)

pmpwith2i committed:
chore: eslint automatic fix

pmpwith2i committed:
fix: lint and format

pmpwith2i commented:
@DonnySolana I updated the code, almost of the 38 files changed are automatically updated by prettier and eslint, I just removed some memo not needed where the file are imported from an external module (because it never changes), like the _**notify**_
